specs/waterOrderingSpec.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const rewiremock = require('./helpers/rewiremock')
+
+const sendgridMock = {
+  send: async (message) => {},
+  setApiKey: (key) => {}
+};
+
+const httpsGetMock = {
+  get: async (url) => {}
+}
+
+rewiremock('@sendgrid/mail').with(sendgridMock)
+rewiremock('https').callThrough().with(httpsGetMock)
+rewiremock.enable()
+const waterOrderingController = require('../features/waterOrdering')
+rewiremock.disable()
+const replies = require('../features/waterOrdering/replies.js')
+const { getBasicController } = require('./helpers')
+
+describe('Water ordering controller', () => {
+  beforeEach(() => {
+    this.controller = getBasicController()
+    waterOrderingController(this.controller)
+  })
+
+  it('Should return any confirmation if user types `mona order water`', async () => {
```

Comment:
please check previous PRs with tests, there is an explanation of why we do not use 'should' in tests

Reply:
@AleksSenkou Yep, I always use `expect` instead of `should` in the ruby tests but don't think that the `RSpec guidelines with Ruby` related to the JS best practices too.

Reply:
@npupko it doesn't really matter, good practice is a good practice 😄